src/lib/layer-manager.js

```diff
@@ -594,6 +594,10 @@ export default class LayerManager {
    *                        {Object} srcEvent:             native JS Event object
    */
   _onPointerMove(event) {
+    if (event.isDown) {
+      // Is dragging
```

Comment:
Can we add a little more to the comment  i.e. why does dragging mean that we don't need move events?